sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/models/RuleProperties.java

```diff
@@ -208,6 +208,12 @@ public RuleFilterImpl toImplementation(RuleFilter model) {
         });
     }
 
+    /**
+     * Initializes a new instance
+     */
+    public RuleProperties() {
```

Comment:
We didn't want to add a public constructor for this so that's why we have the EntityHelper with the EntityRuleAccessor: https://github.com/Azure/azure-sdk-for-java/blob/3ec8fccab55cfaf7428ed9424f57df1bdb4132d6/sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/models/RuleProperties.java#L38